src/pages/node/configuration.tsx

```
@@ -77,50 +78,54 @@ function SettingsPage() {
 
   // Usage in useEffect
   useEffect(() => {
-    if (!strategies || !ticketPrice) return;
+    if (!strategy || !ticketPrice) return;
+
+    let strategyTMP = {hopr: {strategy: JSON.parse(JSON.stringify(strategy))}};
+    delete strategyTMP.hopr.strategy['parsedStrategies']
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)